config/install_smdebug.sh

```diff
@@ -3,11 +3,13 @@ set -ex
 set -o pipefail
 
 
+export SMDEBUG_S3_BINARY="s3://smdebug-nightly-binaries/$(date +%F)/"
 cd $CODEBUILD_SRC_DIR
 # you can provide bip binary as s3 path in the build environment
 if [ "$SMDEBUG_S3_BINARY" ]; then
+  echo "Installing smdebug and smdebug_rules from pre-generated pip wheels at $SMDEBUG_S3_BINARY"
   mkdir -p s3_pip_binary
-  aws s3 cp "$SMDEBUG_S3_BINARY" s3_pip_binary
+  aws s3 cp --recursive "$SMDEBUG_S3_BINARY" s3_pip_binary
```

Comment:
check_smdebug_install - How is this knowing what version to check ? 

$force_check_smdebug_version - this variable controls , see config/check_smdebug_install.sh . We need to make sure force_check_smdebug_version this is set to right version from pip binary.

Reply:
> We need to make sure force_check_smdebug_version this is set to right version from pip binary.

i will do this.